test/github-issues/184/issue-184.ts

```diff
@@ -26,7 +26,7 @@ describe("github issues > #184 [Postgres] Single-Inheritance not working with in
         employee.shared = "e";
 
         await employeeRepository.save(employee);
-        await employeeRepository.findOne(1);
+        await employeeRepository.findOne(employee.id);
```

Comment:
```employee.id``` is a string, not an integer, so Postgres throws an error. Is this supposed to work?